charts/kyverno/templates/cleanup-controller/_helpers.tpl

```diff
@@ -0,0 +1,29 @@
+{{/* vim: set filetype=mustache: */}}
+
+{{- define "kyverno.cleanup-controller.deploymentName" -}}
+cleanup-controller
+{{- end -}}
+
+
+{{- define "kyverno.cleanup-controller.labels" -}}
+app.kubernetes.io/component: cleanup-controller
+app.kubernetes.io/instance: {{ .Release.Name }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+app.kubernetes.io/name: {{ template "kyverno.name" . }}
+app.kubernetes.io/part-of: {{ template "kyverno.name" . }}
+app.kubernetes.io/version: "{{ .Chart.Version }}"
```

Comment:
Do you need `helm.sh/chart: {{ template "kyverno.chart" . }}` here too?

Reply:
I don't think so, it's added by helm right ?

Reply:
I do not believe it is , but I could be wrong. The other templates like main `kyverno.labels` add that label explicitly.

Reply:
Looks like it's a recommended label https://helm.sh/docs/chart_best_practices/labels/#standard-labels